src/Examples/publish-event-with-auth.php

```diff
@@ -22,7 +21,10 @@
     $note->setContent($content);
     // Sign event.
     $private_key = new Key();
-    $private_key = $private_key->generatePrivateKey();
+
+    // pull environment variable containing key that is allowed to AUTH to this relay
+    // uses the same env name as NAK
+    $private_key = getenv('NOSTR_SECRET_KEY');
```

Comment:
If this env variable is not set, I would suggest to fallback to a generated private key (or use a harcoded one like `11111111`)